frontend/coprs_frontend/coprs/templates/coprs/detail/settings/integrations.html

```diff
@@ -82,7 +82,11 @@ <h3>Webhook History</h3>
                 {% for webhook in webhook_history %}
                 <tr>
                     <td  class="webhook_timestamp" data-toggle="tooltip" title="{{webhook.created_on|localized_time(g.user.timezone)}}">{{webhook.created_on|localized_time(g.user.timezone)}} ({{webhook.created_on|time_ago()}})</td>
-                    <td>{{webhook.webhook_uuid}} </td>
+                    <td>{% if webhook.webhook_uuid is none %}
+                        {{"Custom Webhook"}}
```

Comment:
This goes into the "Webhook ID", I'd probably prefer something like "No UUID generated for this webhook call.".

The double-curly-brackets are not needed.

Also, can you change the header like s/Webhook ID/Forge-generated webhook UUID/g?

Reply:
Yeah, I've seen people triggering the GitHub/GitLab webhooks manually.  I did that a few times for debugging purposes:
```
curl -X POST -H "X-GitHub-Event: ..." -H "Content-Type: application/json" --data "$(cat some json)" ...
```

And also in the future, we might add webhooks that will not have forge-generated UUIDs (not just custom hooks).

Reply:
done
![image](https://github.com/user-attachments/assets/32f3b307-4a5c-4aeb-a210-5fd0601693d2)